hardhat/hardhat.config.ts

```diff
@@ -90,6 +90,44 @@ task('create-user-policy', 'Creates a policy for the given user')
     });
   });
 
+task('accept-claim', 'Accepts the given claim')
+  .addParam('claimId', 'The claim ID')
+  .setAction(async (args, hre) => {
+    const accounts = await hre.ethers.getSigners();
+
+    // The index for the manager needs to be in sync with the deploy script
+    const manager = accounts[1];
+    const claimsManager = ClaimsManagerFactory.connect(process.env.REACT_APP_CLAIMS_MANAGER_ADDRESS!, manager);
+    await claimsManager.acceptClaim(BigNumber.from(args.claimId));
+    console.info(`Accepted Claim: ${args.claimId}`);
+  });
+
+task('propose-settlement', 'Proposes a settlement amount for the claim')
+  .addParam('claimId', 'The claim ID')
+  .addParam('amount', 'The settlement amount')
+  .setAction(async (args, hre) => {
+    const accounts = await hre.ethers.getSigners();
+
+    // The index for the manager needs to be in sync with the deploy script
+    const manager = accounts[1];
+    const claimsManager = ClaimsManagerFactory.connect(process.env.REACT_APP_CLAIMS_MANAGER_ADDRESS!, manager);
+    await claimsManager.proposeSettlement(BigNumber.from(args.claimId), parseApi3(args.amount));
```

Comment:
btw. you don't need to convert to BigNumber when calling the contract functions.

Reply:
Good to know thanks 👍